tardis/visualization/widgets/shell_info.py

```diff
@@ -474,9 +474,7 @@ def display(
 
         # Notes text explaining how to interpret tables widgets' data
         text = ipw.HTML(
-            "<b>Frac. Ab.</b> denotes <i>Fractional Abundances</i> (i.e all "
-            "values sum to 1)<br><b>W</b> denotes <i>Dilution Factor</i> and "
-            "<b>Rad. Temp.</b> is <i>Radiative Temperature (in K)</i>"
+            "<span style='font-size: 1.15em; color:white; background-color:black'><b>Frac. Ab.</b> denotes <i>Fractional Abundances</i> (i.e all values sum to 1)<br><b>W</b> denotes <i>Dilution Factor</i> and <b>Rad. Temp.</b> is <i>Radiative Temperature (in K)</i></span>"
```

Comment:
Can we split this over multiple lines (similar to the line it replaces)? We like to have short lines for easy reading in the terminal or an IDE

Reply:
Sure, I'll look into it.